src/main.js

```diff
@@ -119,22 +140,22 @@ function insertTableHeaderCell(row, text) {
  * Shows Module Identifier (Nummer), Credits (ECTS), Module-Type, Numeric Mark (1-6) and Grade (A-F).
  *
  */
-function createModulesTable(div, modules) {
+async function createModulesTable(div, modules) {
```

Comment:
I you wouldn't load the JSON for every call of `i18n.getMessage()`, this method wouldn't need to be async.

Reply:
done